js/index.js

```diff
@@ -21,6 +21,10 @@ for (let i = 0; i < numOfMembers; i++) {
   memberLinks.appendChild(memberTags);
   memberSection.appendChild(memberLinks);
 }
+const joinButtonDisable = document.getElementById('button-disable');
+joinButtonDisable.addEventListener('click', (event) => {
+  event.preventDefault();
+});
```

Comment:
```suggestion
const joinButtonDisable = document.getElementById('button-disable');
joinButtonDisable.disabled = true;
```

Reply:
When we use pointer-events:none ,there is no use of JS code,I guess.